### PR TITLE
Add IPY operation detail

### PR DESCRIPTION
**Pull request by lauragilgz**
We have added IPY Operation detail, this is based in the following documentation:
image

### CODE REVIEW

**Review by frantisekrokusek — APPROVED** [review]

**Review by NathanBerthier — APPROVED** [review]
Nice 👌

**Review by scollon-pl — APPROVED** [review]

File: cfonb.gemspec [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Gem::Specification.new do |s|
   s.name                       = 'cfonb'
   s.version                    = '0.0.7'
-  s.required_ruby_version      = '>= 3.2.2'
+  s.required_ruby_version      = '>= 3.3.5'
```
Comment (RDeckard) [inline cfonb.gemspec]: Is this Ruby bump really mandatory for project requiring this gem?

File: Gemfile [hunk 2]
```diff
@@ -3,7 +3,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '3.2.2'
+ruby '3.3.5'
```
Comment (RDeckard) [inline Gemfile]: I don't think we should specify a Ruby version here, as `cfonb` is just a dependency, not _final product_: the usual directive in a Gemfile for a gem is `gemspec`.

```suggestion
gemspec
```

File: Gemfile.lock
Comment (RDeckard) [inline Gemfile.lock:1]: For the same reason (a dependency is not a _final product_ and should only define minimal specifications, and not a specific runtime), this file should be in a `.gitignore` [I've just created it].

File: .ruby-version
[the remark this answers is not in the document]
Comment (RDeckard) [reply .ruby-version:1]: OK, it looks that our CI need it, let's keep it then!

**Comment by RDeckard** [comment]
Hey @lauragilgz, sorry for the mess, @frantisekrokusek and I talked about this PR and we did some tests but I realize that it might add a bit of confusion here. 🙏 
I'll keep you posted!

**Comment by lauragilgz** [comment]
@RDeckard Sure thing! No problem at all 😄

**Review by RDeckard — APPROVED** [review]
OK, all goods now! ✅

**Pull request merged**